adoc/chapters/programming_interface.adoc

```diff
@@ -4689,7 +4689,8 @@ accessor<T, Dimensions, Mode, Targ> get_access(handler& commandGroupHandler,
       mode and target in the command group buffer.  The accessor is a
       <<ranged-accessor>>, where the range starts at the given offset from the
       beginning of the buffer.  The value of target can be
-      [code]#target::device# or [code]#target::constant_buffer#.
+      [code]#target::device#, [code]#target::constant_buffer# or
+      [code]#target::host_task.
```

Comment:
This is a good catch, but I think we also need to update the description of the overload above (`get_access(handler& commandGroupHandler)`) with the same wording.

Reply:
Ah yes, I missed that. I've now modified that overload's description as well.